src/Common/src/System/IO/PathInternal.Windows.cs

```diff
@@ -53,13 +53,20 @@ internal static partial class PathInternal
 
         internal static readonly char[] InvalidPathChars =
         {
-            '\"', '<', '>', '|', '\0',
```

Comment:
FindFirstFile is where the wildcards come into play- and is why we have the current ones (*, ?) broken out.  We check all characters in GetFullPath.

What does or doesn't work with CreateFile is device/file system dependent. You can use most of these with pipes, for example. Null and backslash are the most "reserved" of the lot as the null terminates string logic and the backslash is the separator the object manager uses.